CONTRIBUTING.md

```diff
@@ -0,0 +1,163 @@
+2. [Clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) the forked repository.
+3. [Configure remotes](https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-original-repository).
+4. Create a feature [branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches)
+   off of the `main` branch.
+   ```
+   git checkout -b <feature-branch-name>
+   ```
+5. Make your changes. Make sure to follow the [Style Guidelines](#style-guidelines). Commit your changes.
+   ```
+   git add -A
+   git commit
+   ```
+   Commit messages should follow the [Commit Message Guidelines](#commit-message-guidelines).
+6. Make sure to update, or add to any tests where appropriate. Try to run tests locally first (`go test ./...` for the
+   `backend`, see the [README](./e2e/README.md) for the `e2e`tests on how to run them).
+7. If you added or changed a feature, make sure to document it in the README.md file. If your change
+   affects the `backend` API update the [Open API spec(s)](./docs/static/spec).
+   If your changes affect the `backend` configuration, update the [Config.md](./backend/docs/Config.md).
+8. Push your feature branch up to your fork:
+   ```
+   git push origin <feature-branch-name>
+   ```
+9. [Create a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+10. Submit the pull request by filling out the [pull request template](./.github/PULL_REQUEST_TEMPLATE.md)
+    (note: the template should be displayed automatically once you open a pull request; take account of the comments in
+    the displayed template).
+11. If a pull request is not ready to be reviewed it should be marked as a "Draft".
+
+
+When pull requests fail test checks, authors are expected to update
+their pull requests to address the failures until the tests pass. If you have trouble or questions on how to add to
+existing tests, reach out through our [communication](#communication) channels.
+
+# Commit Message Guidelines
+
+Commit messages should adhere to the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+The commit message should be structured as follows:
+
+```
+<type>(<optional scope>): <description>
+
+<optional body>
+
+<optional footer(s)>
+```
+
+The commit message headline should have the following structure:
+```
+<type>(<optional scope>): <description>
+│       │             │
+│       │             └─⫸ Summary in present tense. Not capitalized. No period at the end.
+│       │
+│       └─⫸ Commit Scope: optional
+│
+└─⫸ Commit Type: build|ci|docs|feat|fix|perf|refactor|test
```

Comment:
At least on Windows, it seems to me that the arrows do not align correctly:

![image](https://user-images.githubusercontent.com/20115649/196998057-39077e4b-9034-4b43-8dd7-5d0b347b435f.png)

Reply:
It's fine now